airflow/executors/kubernetes_executor.py

```diff
@@ -46,14 +46,14 @@
 from airflow.kubernetes.kube_config import KubeConfig
 from airflow.kubernetes.kubernetes_helper_functions import annotations_to_key, create_pod_id
 from airflow.kubernetes.pod_generator import PodGenerator
-from airflow.models.taskinstance import TaskInstance
```

Comment:
Do we need a local import of this in the [_change_state](https://github.com/apache/airflow/blob/d4dc73416a652a895ada6484d67879a1aad5f0bb/airflow/executors/kubernetes_executor.py#L746) method for the [line](https://github.com/apache/airflow/blob/d4dc73416a652a895ada6484d67879a1aad5f0bb/airflow/executors/kubernetes_executor.py#L776) ?

Reply:
Test failures say yes!